robotoff/cli/main.py

```diff
@@ -1327,5 +1327,53 @@ def launch_normalize_barcode_job(
                 logger.info("Updated %d images", updated)
 
 
+@app.command()
+def update_insight_with_image_field() -> None:
+    """Refresh the `with_image` field of all insights."""
+    import tqdm
+
+    from robotoff.models import ImageModel, ProductInsight, db
+    from robotoff.types import InsightType
+    from robotoff.utils.logger import get_logger
+
+    get_logger()
+
+    with db.connection_context():
+        with db.atomic() as tsx:
+            for insight in tqdm.tqdm(
+                ProductInsight.select()
+                .where(
+                    ProductInsight.type.in_(
+                        [InsightType.brand.name, InsightType.category.name]
+                    ),
+                    # Only non-annotated insights
+                    ProductInsight.annotation.is_null(True),
+                )
+                .iterator(),
+                desc="insight",
+            ):
+                has_image = (
```

Comment:
I'm just wondering if it would be more interesting to directly save the image id instead of a boolean, such that latter you don't need to fetch product information to display `source_image`